src/go/k8s/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,8 @@
+// Copyright 2021 Vectorized, Inc.
```

Comment:
i think we should move this to `/licenses/boilerplate.bsl.txt`

Reply:
you can add a symlink if you need to.

Reply:
Done